### PR TITLE
[WIP] Added BeginText, EndText, and TextOut

### PR DESCRIPTION
**Pull request by Ortuna**
Still adding a few calls that I need for a personal project. What else will I need to do to get the final PR merged?

### CODE REVIEW

**Comment by rosylilly** [comment]
Please write tests and add `C.free(unsafe.Pointer(cString))`

**Comment by Ortuna** [comment]
@rosylilly Could I bother you to check if I'm heading in the right direction?

File: font.go [hunk 1]
```diff
@@ -43,7 +43,6 @@ func (pdf *PDF) LoadType1FontFromFile(afmFn string, pfmFn ...string) (string, er
 	}
 
 	if fontName != nil {
-		defer C.free(unsafe.Pointer(fontName))
```
Comment (rosylilly) [inline font.go]: Thank you so much :bow:

**Comment by rosylilly** [comment]
LGTM

**Comment by Ortuna** [comment]
@rosylilly I think I've added what I needed for now (will send additional PRs when I get more time). This PR should fix #6. This PR is ready for review.

**Pull request merged**